data/net/minecraft/world/level/levelgen/feature/treedecorators/AlterGroundDecorator.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/world/level/levelgen/feature/treedecorators/AlterGroundDecorator
+	METHOD <init> (Lnet/minecraft/world/level/levelgen/feature/stateproviders/BlockStateProvider;)V
+		ARG 1 provider
```

Comment:
Even though `provider` is the field name, this is consistent with all other state providers
```suggestion
		ARG 1 stateProvider
```

Reply:
Would be good to know what anybody from the Parchment team says, e.g. @Lanse505